### PR TITLE
add term dysplasia of the proximal femoral epiphyses

### PR DESCRIPTION
**Pull request by twhetzel**
closes https://github.com/monarch-initiative/mondo/issues/7379

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -570254,6 +570257,14 @@ relationship: has_characteristic MONDO:0021152 ! inherited
 property_value: http://purl.org/dc/terms/creator "https://orcid.org/0000-0002-3458-4839" xsd:string
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6897" xsd:anyURI
 
+[Term]
+id: MONDO:1030002
+name: dysplasia of the proximal femoral epiphyses
+def: "A developmental disorder affecting the growth and development of the proximal end of the femur (thigh bone) near the hip joint characterized by avascular necrosis of the femoral head, cystic changes of the femoral head, and/or sclerosis of the femoral head. It is a relatively milder form of the other skeletal disorders associated with COL2A1." [https://orcid.org/my-orcid?orcid=0009-0003-2127-3550]
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:570263]: This orcid is not valid. The requester probably copied/pasted incorrectly. 
In addition to updating this orcid, you can add the ClinGen working group URL

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -570254,6 +570257,14 @@ relationship: has_characteristic MONDO:0021152 ! inherited
...
+is_a: MONDO:0022800 ! type 2 collagenopathy
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:570264]: source is missing for this SubClassOf

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -160229,6 +160229,7 @@ xref: SCTID:270545000 {source="DOID:14415"}
 xref: UMLS:C0023234 {source="MEDGEN:6035", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 is_a: MONDO:0018381 {source="DOID:14415", source="GARD:0006874", source="MONDO:Redundant", source="Orphanet:2380"} ! osteochondrosis
 is_a: MONDO:0022800 {source="PMID:31633310"} ! type 2 collagenopathy
+is_a: MONDO:1030002 ! dysplasia of the proximal femoral epiphyses
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:160232]: source is missing for this SubClassOf

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -257459,6 +257460,7 @@ xref: SCTID:254064009 {source="MONDO:equivalentTo"}
 xref: UMLS:C0432214 {source="MEDGEN:609409", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 is_a: MONDO:0016761 {source="Orphanet:93279"} ! spondyloepiphyseal dysplasia
 is_a: MONDO:0022800 {source="Orphanet:93279", source="PMID:31633310"} ! type 2 collagenopathy
+is_a: MONDO:1030002 ! dysplasia of the proximal femoral epiphyses
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:257463]: source is missing for this SubClassOf

File: src/ontology/mondo-edit.obo [hunk 5]
```diff
@@ -526837,6 +526839,7 @@ xref: Orphanet:86820 {source="OMIM:608805"}
 xref: UMLS:C4551562 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:1639295"}
 is_a: MONDO:0012126 {source="OMIM:608805"} ! familial avascular necrosis of femoral head
 is_a: MONDO:0022800 {source="PMID:31633310"} ! type 2 collagenopathy
+is_a: MONDO:1030002 ! dysplasia of the proximal femoral epiphyses
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:526842]: source is missing for this SubClassOf

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@twhetzel 
Please see comments. There are some source missing, and incorrect orcid.

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:570263]: Thank you, fixed now.

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:570264]: fixed

File: src/ontology/mondo-edit.obo (hunk 3 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:160232]: fixed

File: src/ontology/mondo-edit.obo (hunk 5 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:526842]: fixed

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:570264]: @twhetzel I still don't see a source for this SubClassOf axiom

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@twhetzel There is still a missing source

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:570264]: it's really added this time!

**Review by sabrinatoro — APPROVED** [review]
Approving and Merging! Thank you @twhetzel

**Pull request merged**